translate/translate.go

```diff
@@ -211,8 +211,27 @@ type TranslationSet struct {
 	Set     map[string]Translation
 }
 
+func NewTranslationSet(fromTag, toTag string) TranslationSet {
+	return TranslationSet{
+		FromTag: fromTag,
+		ToTag:   toTag,
+		Set:     map[string]Translation{},
+	}
+}
+
+func (ts TranslationSet) String() string {
+	str := fmt.Sprintf("from: %v\nto: %v\n", ts.FromTag, ts.ToTag)
+	for k, v := range ts.Set {
+		str += fmt.Sprintf("%s: %v -> %v\n", k, v.From.String(), v.To.String())
```

Comment:
Optional: Might be useful to tab these entries so you know they belong to this specific `TranslationSet`.

Reply:
I think I'll pass. That implies their under the from/to tags which isn't really accurate. It's only added for debugging so I don't think it's too important.